### DISCUSSION TITLE
Add support for pulling Geofence data from Supercharge.info

### DISCUSSION THREAD

**Discussion by MP3Mike** [teslamate-org/teslamate#2716] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

When visiting a new Supercharger site it would be nice if it could be identified from the data at Supercharge.info and create the geofence data automatically. (Pricing isn't available there but it would be easy enough to edit it ourselves.)

Alternatively, you could have an option to pull Supercharge.info data into the geofence database either on demand or on a schedule. (Maybe only for a selected region.)

They do have an API available: https://forum.supercharge.info/t/api-endpoint-for-all-data-of-a-particular-site/1433

### Relevant log output

```shell
N/A
```

### Type of installation

Docker

### Version

v1.26.1

**Comment by DrMichael** [comment]
What information would you possibly want to have? If the current price could be get, that would be terrific. But so...

**Comment by ThomasGorisse** [comment]
SceneView v4 is now **100% Jetpack Compose** — no XML, no fragments.

```kotlin
SceneView(
    modifier = Modifier.fillMaxSize(),
    engine = rememberEngine(),
    modelLoader = rememberModelLoader(engine)
) {
    val model = rememberModelInstance(modelLoader, "models/character.glb")
    if (model != null) ModelNode(modelInstance = model)
}
```

Get started:
```kotlin
implementation("io.github.sceneview:sceneview:4.0.0-rc.1")
```

Full API reference in [llms.txt](https://github.com/sceneview/sceneview/blob/main/llms.txt).
